client/ayon_aftereffects/api/lib.py

```diff
@@ -3,6 +3,7 @@
 import json
 import contextlib
 import logging
+import pyblish
```

Comment:
Not sure if this works all the time, but works in console from tray so I guess it might be ok.